src/src_V2/components/loading-spinner/LoadingSpinner.module.css

```diff
@@ -0,0 +1,78 @@
+@import '../../styles/colors.css';
+
+.preloader {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+}
+
+.loader {
+  display: block;
+  position: relative;
+  left: 50%;
+  top: 50%;
+  width: 150px;
+  height: 150px;
+  margin: -75px 0 0 -75px;
+  border-radius: 50%;
+  border: 3px solid transparent;
+  border-top-color: var(--primary);
+  -webkit-animation: spin 2s linear infinite;
+  animation: spin 2s linear infinite;
+}
+
+.loader:before {
+  content: '';
+  position: absolute;
+  top: 5px;
+  left: 5px;
+  right: 5px;
+  bottom: 5px;
+  border-radius: 50%;
+  border: 3px solid transparent;
+  border-top-color: var(--primary);
+  -webkit-animation: spin 3s linear infinite;
+  animation: spin 3s linear infinite;
+}
+
+.loader:after {
+  content: '';
+  position: absolute;
+  top: 15px;
+  left: 15px;
+  right: 15px;
+  bottom: 15px;
+  border-radius: 50%;
+  border: 3px solid transparent;
+  border-top-color: var(--primary);
+  -webkit-animation: spin 1.5s linear infinite;
+  animation: spin 1.5s linear infinite;
+}
+
+@-webkit-keyframes spin {
+  0% {
+    -webkit-transform: rotate(0deg);
+    -ms-transform: rotate(0deg);
+    transform: rotate(0deg);
+  }
+  100% {
+    -webkit-transform: rotate(360deg);
+    -ms-transform: rotate(360deg);
+    transform: rotate(360deg);
+  }
+}
+
+@keyframes spin {
+  0% {
+    -webkit-transform: rotate(0deg);
+    -ms-transform: rotate(0deg);
+    transform: rotate(0deg);
+  }
+  100% {
+    -webkit-transform: rotate(360deg);
+    -ms-transform: rotate(360deg);
+    transform: rotate(360deg);
+  }
+}
```

Comment:
@aninaslyan jan i saw the work you did, but the same question comes in my mind, don't we have this component in mui? I saw there are many loaders in mui.

Reply:
@igortokhunts jan you are right, mui has many loading indicators. This is the loading design that our designers wanted to have in the whole application. They sent me [the link](https://codepen.io/WebSonata/pen/bRaONB).
But the design is not final. As they wanted to have a different design and in the future, they want to have a fixed overlay so nothing behind is clickable (they promised to create figma screen). That's why I agreed to use the custom design, not the material one.

Reply:
If this loader we need to use in the entire app it's better to have it in the components folder, am i right? is it global for entire app?

Reply:
It's already in components folder. Yes, it's for the entire app.

Reply:
great